pkg/build/node.go

```diff
@@ -155,9 +158,11 @@ func node(root string, options KindOptions) (string, error) {
 		{{end}}
 
 		{{if .IsYarn}}
-		RUN yarn --non-interactive
+		RUN yarn --non-interactive --production --frozen-lockfile
```

Comment:
minor but while you're here, this seems clearer:

```suggestion
		RUN yarn install --non-interactive --production --frozen-lockfile
```